hnn_core/gui/_viz_manager.py

```diff
@@ -716,20 +755,31 @@ def __init__(self, gui_data, viz_layout):
             (self.figs_tabs, 'selected_index'),
         )
 
-        template_names = list(fig_templates.keys())
+        template_names = list(data_templates.keys())
+        template_names.extend(list(fig_templates.keys()))
         self.templates_dropdown = Dropdown(
             description='Layout template:',
             options=template_names,
             value=template_names[0],
             style={'description_width': 'initial'},
             layout=Layout(width="98%"))
+        self.templates_dropdown.observe(self.layout_template_change, 'value')
+
         self.make_fig_button = Button(
             description='Make figure',
             button_style="primary",
             style={'button_color': self.viz_layout['theme_color']},
             layout=self.viz_layout['btn'])
         self.make_fig_button.on_click(self.add_figure)
 
+        self.datasets_dropdown = Dropdown(
+            description='Dataset:',
+            options=[],
+            value=None,
+            style={'description_width': 'initial'},
+            layout=Layout(width="98%"))
+        self.datasets_dropdown.layout.visibility = "hidden"
```

Comment:
Since Dipole-Drive is the new default plot at the start I think the visibility should be "visible" and only hidden if a blank template is selected.

Reply:
I set the default visibility as `visible` as a default